### PR TITLE
Multi uc

### PR DESCRIPTION
**Pull request by Jozefiel**

### CODE REVIEW

File: config/krakend/krakend.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     {{ template "workflow_proxy.tmpl" .workflow_proxy_settings.endpoints }},
     {{ template "workflow_proxy.tmpl" .workflow_proxy_settings_deprecated.endpoints }},
     {{ template "uniconfig.tmpl" .uniconfig_settings }},
-    {{ template "uniconfig_deprecated.tmpl" .uniconfig_settings_deprecated }},
```
Comment (marosmars) [inline config/krakend/krakend.json:35]: have the deprecated settings been removed ?

File: startup.sh [hunk 2]
```diff
@@ -11,10 +11,12 @@ DESCRIPTION:
   - If no options are specified, starts UniFlow and UniConfig services on local single node 
     with development resources allocation and http protocol.
 
-  - For starting FM in multi-node, change value of UC_SWARM_NODE_ID in .env file and use
-    --uniflow-only for deploy Uniflow on swarm manager node and
-    --deploy-uniconfig for deploy Uniconfig on swarm worker node.
-
+  - For starting FM in multi-node is need to generate uniconfig compose files with script:
+    ./generate_uc_compose.sh --service-name <arg> --node-id <arg> --folder-path <arg>
+    //TODO
```
Comment (marosmars) [inline startup.sh:16]: is there a leftover here for us ?

**Review by Jozefiel — COMMENTED** [review]
help output is updated

File: config/krakend/krakend.json (hunk 1 above)
Comment (marosmars) [reply config/krakend/krakend.json:35]: That's why I am asking ... whether this file should be removed from the codebase:

./config/krakend/settings/uniconfig_settings_deprecated.json

**Pull request merged**